### PR TITLE
Update Test Visibility docs: add links to Github Action

### PR DESCRIPTION
**Pull request by nikita-tkachenko-datadog**
### What does this PR do? What is the motivation?

This PR updates Test Visibility setup docs for .NET, Java, JS and Python.
A link to Test Visibility Github Action is added for this languages.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tests/setup/java.md [hunk 1]
```diff
@@ -66,26 +62,26 @@ Other build systems, such as Ant or Bazel, are supported with the following limi
...
 ## Setup
 
-Setting up Test Visibility for Java includes the following steps:
-1. Configure tracer reporting method.
-2. Download tracer library to the hosts where your tests are executed.
-3. Run your tests with the tracer attached.
+You may follow interactive setup steps on the [Datadog site][2] or the instructions below.
 
-You may follow interactive setup steps on the [Datadog site][2] or with the instructions below.
+Configuring Datadog Java Tracer varies depending on your CI provider.
```
Comment (urseberry) [inline content/en/tests/setup/java.md:67]:
```suggestion
Configuring the Datadog Java Tracer varies depending on your CI provider.
```

File: content/en/tests/setup/java.md [hunk 2]
```diff
@@ -66,26 +62,26 @@ Other build systems, such as Ant or Bazel, are supported with the following limi
...
 
-### Configuring reporting method
+{{< tabs >}}
+{{% tab "Github Actions" %}}
+You can use the dedicated <a href="https://github.com/marketplace/actions/configure-datadog-test-visibility">Datadog Test Visibility Github Action</a> to enable Test Visibility.
+If you do so, you can skip the "Downloading tracer library" and "Running your tests" steps below.
```
Comment (urseberry) [inline content/en/tests/setup/java.md:72]:
```suggestion
If you do so, you can skip the **Downloading tracer library** and **Running your tests** steps below.
```

**Review by urseberry — CHANGES_REQUESTED** [review]
Can you change the links to use numbered links instead of HTML links? Our translation process does not support HTML links. Here is some documentation on how to properly add links within a tab in the public documentation: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Tabs

**Comment by nikita-tkachenko-datadog** [comment]
> Can you change the links to use numbered links instead of HTML links? Our translation process does not support HTML links. Here is some documentation on how to properly add links within a tab in the public documentation: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Tabs

Thanks for the explanation, replaced the HTML links with the numbered links.

**Review by urseberry — APPROVED** [review]

**Pull request merged**